ccgains/bags.py

```diff
@@ -176,10 +176,16 @@ def __init__(
         :param relation:
             A CurrencyRelation object which serves exchange rates
             between all currencies involved in trades which will later
-            be added to this BagFIFO.
+            be added to this BagQueue.
+
+        :param mode:
+            Inventory accounting method to use. The following methods are supported:
+            - FIFO: First In First Out
+            - LIFO: Last In First Out
+            - LPFO: Lowest Price First Out
```

Comment:
Maybe rather write it this way (adding "): `- "FIFO": First In First Out`, then it is clear that it must be a string

Reply:
Agree, I'll also add explicit type indication `:param mode: (string)`